policy/modules/system/systemd.te

```diff
@@ -1,5 +1,9 @@
 policy_module(systemd, 1.10.4)
 
+gen_require(`  #selint-disable:S-001
+        attribute daemon;
```

Comment:
This isn't acceptable. I think what probably makes sense is to create a new systemd interface that has `allow $1 systemd_socket_proxyd_t:unix_stream_socket connectto;`, and then put a call to that new interface in an optional in the systemd block of init_domain().

Reply:
I took it from https://github.com/SELinuxProject/refpolicy/blob/master/policy/modules/system/selinuxutil.te#L3 (and there are at least two more places that uses the same: `domain.te` and `corenetwork.te.m4`).  Anyway, I will think how to resolve it properly (using your advice)